concourse-production.yml

```diff
@@ -28,3 +28,10 @@ instance_groups:
   azs: [z1]
   networks:
   - name: production-concourse
+- name: iaas-worker
+  vm_type: production-concourse-worker
+  vm_extensions: [production-concourse-iaas-profile]
+  instances: 1
```

Comment:
Looks like workers are `m3.xlarge`, so we can probably get away with `m3.large` or even `m3.medium`.